unified-codes/libs/ui/src/lib/components/entity/molecules/EntityDetailCategoryItem.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { IEntity } from '@unified-codes/data';
+
+export interface EntityDetailCategoryItemProps {
+  entity: IEntity;
+}
+
+export type EntityDetailCategoryItem = React.FunctionComponent<EntityDetailCategoryItemProps>;
```

Comment:
almost getting too long as a name.. would `EntityCategory` be descriptive enough?

Reply:
I do agree it's getting too long 😞  - another case of struggling with names. I was thinking `Detail` would help denote it as specific to `EntityDetails` and not related to the `EntityTable` used in the explorer. Would `EntityDetailCategory` work? 🤔  Still a little long

Reply:
I think when the names for components start getting long it's usually an indication of thinking in OOP inheritance instead of composition (as opposed to big long interface names, which are quite common, I think because of OOP's use of inheritance)! I think this component may be a bit too specific, could probably handle the list logic in the parent (possibly using a generic presentational component to display it)?